### PR TITLE
Customer MP: Chat availability

### PR DESCRIPTION
**Pull request by bstien**
# Why?
For the Customer MP we need a new view to let the users connect to the dealerships video chat, as well as seeing their online status.

# What?
- Added `ChatAvailabilityView` and demo.

# Show me
| Still image | Status switching |
| --- | --- |
| Screenshot 2020-04-03 at 12 23 09 | Kapture 2020-04-03 at 12 27 47 |

### CODE REVIEW

**Comment by bstien** [comment]
Not sure what's going on with the snapshot tests. This is the diff for `testLoginView()`. The others are similar to this 🤔

Screenshot 2020-04-03 at 12 41 12

File: Demo/Demo/Components/LoadingIndicator/LoadingIndicatorViewDemoView.swift [hunk 1]
```diff
@@ -5,17 +5,8 @@
...
-        return view
-    }()
-
-    private lazy var delayedLoadingIndicatorView: LoadingIndicatorView = {
-        let view = LoadingIndicatorView()
-        view.translatesAutoresizingMaskIntoConstraints = false
-        return view
-    }()
+    private lazy var loadingIndicatorView = LoadingIndicatorView(withAutoLayout: true)
+    private lazy var delayedLoadingIndicatorView = LoadingIndicatorView(withAutoLayout: true)
```
Comment (fespinoza) [inline Demo/Demo/Components/LoadingIndicator/LoadingIndicatorViewDemoView.swift:9]: ❤️

File: FinnUI/Sources/Components/ChatAvailability/ChatAvailabilityView.swift [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+            chatNowButton.isEnabled = false
+        }
+
+        statusView.configure(status: status, statusTitle: statusTitle)
+    }
+
+    // MARK: - Private methods
+
+    @objc private func handleButtonTap() {
+
```
Comment (fespinoza) [inline FinnUI/Sources/Components/ChatAvailability/ChatAvailabilityView.swift]: is there something missing here? a delegate/callback call, etc

Comment (bstien) [reply FinnUI/Sources/Components/ChatAvailability/ChatAvailabilityView.swift]: Hehe, forgot it during the all hands meeting😄 Added it a couple of minutes ago, though💪

**Pull request merged**